dbt_project.yml

```diff
@@ -871,6 +871,8 @@ models:
       +schema: yield_yak
       avalanche_c:
         +schema: yield_yak_avalanche_c
+      arbitrum:
+        +schema: yield_yak_arbitrum
```

Comment:
can you remove modifications to this file and instead put schema property directly in model config blocks?

Reply:
Done